vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -283,53 +294,50 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
             ]
         )
 
-    def _arrange_page_divs(self, page_divs: _PageDivsType):
+    def _get_outer_page_divs(self, inner_page_divs: _InnerPageDivsType) -> _OuterPageDivsType:
```

Comment:
When you want to make these public I'd go for something simpler like just `outer_page` as the function name - so no `get` or `divs`. e.g. we'd have `custom_header`, `outer_page`, `inner_page`, `arrange_page`.